File: unit2/arg_main.py
import sys

##Variables Globales##
my_int = 0
my_str = ""
my_float = 0.0
my_bool = True ##False##
my_list = [] ##list ()##


if __name__ == "__main__":

    my_int = int(sys.argv[1])
    my_str = str(sys.argv[2])
    my_float = float(sys.argv[3])
    my_bool = bool(sys.argv[4]) ##False##
    my_list = sys.argv
    ##         0,1,2,3,4        ## 
    print(my_int * 2, my_str * 2, my_float, my_bool, "|"  , my_list)

File: unit2/arg_main.c
#include <stdio.h>



int my_int = 0; //%i o %d
float my_float = 3.14; // %f
char my_char = 'c'; //%c
char my_str[10] = "hola"; //%s



float second_float = 0.0;

//Pointer//
int * my_prt_int = &my_int; //%p



int main(int argc, char** argv){

    strcpy(my_str, argv[1]);
    int num = atoi(argv[1]);
    printf("num: %i", num)

    printf("myint: %i , my_float: %f , my_char: %c , my_str: %s , my_ptr_int: %p.\n", my_int, my_float, my_char, my_str, my_prt_int);
    printf("argc: %i, element 1: %s, elem2: %s char. %c \n", num, argv[2], 'J');

    return 0;
}